carto/sql.py

```diff
@@ -444,10 +444,11 @@ def copyto(self, query):
         """
         url = self.api_url + '/copyto'
         params = {'api_key': self.api_key, 'q': query}
+        http_method = 'GET' if len(query) < MAX_GET_QUERY_LEN else 'POST'
```

Comment:
Is it possible for `params` to be too large for a GET if query is just under `MAX_GET_QUERY_LEN`?

Reply:
nginx and other servers are returning the 414 error if the request is longer than 4kb or 8kb. As we are using POST from 1Kb we are secure enough.

> One question is whether the http_method should be exposed as something the user has control over.

I don't think so. For the user, there is no difference between using GET or POST (I started doing that, but I realized it is needless)

Reply:
👍